packages/info/README.md

```diff
@@ -32,18 +32,6 @@ npx webpack info [options]
 
 _Not supported for config_
 
-#### Options
-
-| Flag                | Description                                                     | Type        |
-| ------------------- | --------------------------------------------------------------- | ----------- |
-| `--help`            | Show help                                                       | [ boolean ] |
-| `--version`         | Show version number of `webpack-cli`                            | [ boolean ] |
```

Comment:
help and version are still supported right?

Reply:
Yes, `help` and `version` still supported